.travis.yml

```diff
@@ -1,7 +1,13 @@
 language: node_js
-script: 'if [ -n "$ARTIFACT" ]; then make test; else make test-sauce; fi'
 node_js:
   - '0.12'
+install:
+  - mkdir travis-phantomjs
+  - wget https://s3.amazonaws.com/travis-phantomjs/phantomjs-2.0.0-ubuntu-12.04.tar.bz2 -O $PWD/travis-phantomjs/phantomjs-2.0.0-ubuntu-12.04.tar.bz2
+  - tar -xvf $PWD/travis-phantomjs/phantomjs-2.0.0-ubuntu-12.04.tar.bz2 -C $PWD/travis-phantomjs
+  - export PATH=$PWD/travis-phantomjs:$PATH
```

Comment:
This loads phantomjs 2.0 in travis. This is important because phantomjs < 2.0 does not support `Function.prototype.bind`. Travis should update to support 2.0 soon.